beancount/plugins/close_tree.py

```diff
@@ -0,0 +1,64 @@
+"""This plugin inserts close directives for all of an account's descendants when an account is
+closed. Unopened parent accounts can also be closed. Any explicitly specified close is left
+untouched.
+
+For example, given this:
+
+```
+2017-11-10 open Assets:Brokerage:AAPL
+2017-11-10 open Assets:Brokerage:ORNG
+2018-11-10 close Assets:Brokerage  ; this account does not necessarily need to be opened
+```
+
+the plugin turns it into:
+```
+2017-11-10 open Assets:Brokerage:AAPL
+2017-11-10 open Assets:Brokerage:ORNG
+2018-11-10 close Assets:Brokerage:AAPL
+2018-11-10 close Assets:Brokerage:ORNG
+```
+
+Invoke this plugin _after_ any plugins that generate `open` directives for account trees that you
+want to auto close. An example is the `auto_accounts` plugin that ships with Beancount:
+
+```
+plugin "beancount.plugins.auto_accounts"
+plugin "beancount.plugins.close_tree"
+```
+"""
```

Comment:
Documentation strings do not use Markdown syntax. If anything it should be reST.